gradle/root/coverage.gradle

```diff
@@ -1,16 +1,19 @@
 apply plugin: 'jacoco'
-
-// TODO later include subprojects, possible ideas https://gist.github.com/aalmiray/e6f54aa4b3803be0bcac
+apply from: 'gradle/dependencies.gradle'
 
 jacoco {
-    toolVersion = "0.7.9"
+    toolVersion = libraries.jacocoVersion
 }
 
-task mockitoCoverage(type: JacocoReport, dependsOn: "test") {
-    executionData files("${buildDir}/jacoco/test.exec")
+task mockitoCoverage(type: JacocoReport, dependsOn: ["test", "kotlinTest:test", "extTest:test", "inline:test"]) {
+    executionData fileTree(project.rootDir.absolutePath).include("**/build/jacoco/*.exec")
 
     sourceSets sourceSets.main
 
+    subprojects {
```

Comment:
I looked at the current "mockitoCoverage" implementation and I think that the change is a bit more involved. For example, we should also include ```classDirectories``` from the subprojects.